src/contributing.md

```diff
@@ -116,32 +90,76 @@ make a documentation change, add
 to the end of the pull request description, and [@rust-highfive][rust-highfive] will assign
 [@steveklabnik][steveklabnik] instead of a random person. This is entirely optional.
 
+In addition to being reviewed by a human, pull requests are automatically tested
+thanks to continuous integration (CI). Basically, every time you open and update
+a pull request, the CI builds the compiler and tests it against the
```

Comment:
nit: CI is a process, not a noun
```suggestion
a pull request, CI builds the compiler and tests it against the
```
or
```suggestion
a pull request, Github Actions builds the compiler and tests it against the
```

Reply:
I think I've seen occurences of "the CI" in other parts of the guide... Not sure that's worth fixing, I'll just change that for my PR.